controllers/front/validation.php

```diff
@@ -237,6 +238,28 @@ public function postProcess()
         //Tools::redirectLink(Tools::getHttpHost(true, true) . __PS_BASE_URI__ .'index.php?controller=order-confirmation?id_cart='.(int)($cart->id).'&id_module='.(int)($blockonomics->id).'&id_order='.$blockonomics->currentOrder.'&key='.$customer->secure_key);
     }
 
+    private function getTimeRemaining($order)
+    {
+        if ($order) {
+            $time_remaining = ($order['timestamp'] +
+            (Configuration::get('BLOCKONOMICS_TIMEPERIOD') * 60) - time()) / 60;
+            if ($time_remaining > 0) {
+                return $time_remaining;
+            }
+        }
+        return false;
+    }
+
+    private function getBits($blockonomics, $currency, $total)
```

Comment:
get_bits has been renamed and requires fixing:
![127-0-0-1-prestashop3-index-php-fc-module-module-blockonomics-controller-validation-id_lang-2](https://user-images.githubusercontent.com/39666372/113707738-9d402700-96e0-11eb-838b-d7ad1fb8de22.png)

Reply:
Yes. Now I've corrected both get_bits and get_remaining_time. Both when defining and calling them.